### PR TITLE
FIX: Strip-off node-mongo-script from api-boilerplate template

### PR DESCRIPTION
**Pull request by samuko-things**
**This pull request makes the following changes**
* Fixes collabo-community/contributor-issue-report#5

**General checklist**
- [x] File or folder now contains changes as specified in the issue i worked on
- [x] I have linked the issue I worked on to this pull request submitted by me

**Testing checklist**
- [ ] node-mongo-script dependency has been removed totally from the ts, esm, and cjs boilerplate template
- [ ] the boilerplate templates still works as before but without the script dependency. 
- [ ] I certify that I ran my checklist

Ping @Ifycode @collabo-community/review-backend-api

### CODE REVIEW

**Comment by samuko-things** [comment]
@Ifycode, @collabo-community/review-backend-api 
- I have removed the node-mongo-script from the ts template.
- the ts template still works as before (without the node-mongo-script)
> run ts template with `npm run dev:atlas` or `npm run dev:local`

**Comment by samuko-things** [comment]
- I have removed the node-mongo-script from the **esm** template.
- the **esm** template still works as before (without the node-mongo-script)
> run **esm** template with `npm run dev:atlas` or `npm run dev:local`

**Comment by samuko-things** [comment]
- I have removed the node-mongo-script from the **cjs** template.
- the **cjs** template still works as before (without the node-mongo-script)
> run **cjs** template with `npm run dev:atlas` or `npm run dev:local`

File: cjs/package.json [hunk 1]
```diff
@@ -15,7 +14,6 @@
   "author": "Obiagba Mary",
```
Comment (Ifycode) [inline cjs/package.json:14]: remove `dev:restore` and `dev:change` scripts in the 3 templates

**Review by Ifycode — CHANGES_REQUESTED** [review]
In the `node-mongo-helpers.js` file, only retain the console.log functions: success, warning and error.
All others, you can remove them as those are for the scripts.

> All 3 templates.

File: cjs/package.json (hunk 1 above)
Comment (Ifycode) [reply cjs/package.json:14]: Just remove this one, leave the cancelled comment for me to handle @samuko-things

Comment (Ifycode) [inline cjs/package.json:14]: @samuko-things 
1. remove `dev:restore` and `dev:change` scripts in the 3 templates

File: esm/node-mongo-helpers.js [hunk 2]
```diff
@@ -1,6 +1,9 @@
 import { spawn } from 'child_process';
 import package_json from './package.json';
 import chalk from 'chalk';
+import dotenv from 'dotenv';
+
+dotenv.config();
```
Comment (Ifycode) [inline esm/node-mongo-helpers.js:7]: 2. I have fixed some things in the esm template and I've merged it into this PR.
The changes I made:
- There is now only one `.server` file, and one `.connect` file (the name of the files change accordingly, and I also updated their content accordingly)
- I merged some functions together in the process to achieve this (and the names of the functions change accordingly)
- Added new piece of code in the `node-mongo-helpers.js` to check which connection type the user chose based on which npm dev script they are running, and apply .env settings accordingly.
- Adjusted package.json to single server.js file

Referencing my PR: https://github.com/collabo-community/backend-api-boilerplates/pull/2/files

You are to help to do this exact same thing in the ts and cjs templates @samuko-things

**Review by Ifycode — CHANGES_REQUESTED** [review]

File: cjs/package.json (hunk 1 above)
Comment (samuko-things) [reply cjs/package.json:14]: Ok, I understand @Ifycode

**Comment by samuko-things** [comment]
I have just removed the `dev:change` and `dev:restore` scripts from the 3 templates

File: cjs/package.json (hunk 1 above)
Comment (samuko-things) [reply cjs/package.json:12]: > @samuko-things
> 
> 1. remove `dev:restore` and `dev:change` scripts in the 3 templates

I have just removed the `dev:change` and `dev:restore` scripts from the 3 templates

File: esm/node-mongo-helpers.js (hunk 2 above)
Comment (samuko-things) [reply esm/node-mongo-helpers.js:7]: > 2. I have fixed some things in the esm template and I've merged it into this PR.
>    The changes I made:
> 
> * There is now only one `.server` file, and one `.connect` file (the name of the files change accordingly, and I also updated their content accordingly)
> * I merged some functions together in the process to achieve this (and the names of the functions change accordingly)
> * Added new piece of code in the `node-mongo-helpers.js` to check which connection type the user chose based on which npm dev script they are running, and apply .env settings accordingly.
> * Adjusted package.json to single server.js file
> 
> Referencing my PR: https://github.com/collabo-community/backend-api-boilerplates/pull/2/files
> 
> You are to help to do this exact same thing in the ts and cjs templates @samuko-things

I have made the necessary changes to the ts and cjs template. I have also tested it and it all works.

File: cjs/.env.example [hunk 3]
```diff
@@ -1,9 +1,9 @@
 
-PORT_ATLAS=8090
+PORT_ATLAS=8080
 MONGODB_ATLAS_URI=
 
-PORT_LOCAL=8091
+PORT_LOCAL=8081
```
Comment (Ifycode) [inline cjs/.env.example:5]: `PORT_ATLAS` & `PORT_LOCAL` values in the env.json file I see you changed it in the cjs template. If you change it here, you have to also change the ts template too.

File: cjs/node-mongo-helpers.js [hunk 4]
```diff
@@ -37,13 +69,8 @@ const eslintAndServer = (serverPort) => {
...
 
-const afterAtlasDBconnectSuccessful = (serverPort) => {
-  success('\nConnected to mongoDB ATLAS');
-  eslintAndServer(serverPort);
-}
-
-const afterLocalDBconnectSuccessful = (serverPort) => {
-  success('\nConnected to LOCAL mongoDB');
+const afterDBconnectSuccessful = (serverPort) => {
+  success('\nConnected to mongoDB');
```
Comment (Ifycode) [inline cjs/node-mongo-helpers.js:73]: Add a check for this success string: 
`'\nConnected to [local] mongoDB [Atlas]'`
Where local text shows up only if user is running local, and atlas shows up only if user is running atlas.

**Review by Ifycode — CHANGES_REQUESTED** [review]

File: cjs/node-mongo-helpers.js (hunk 4 above)
Comment (Ifycode) [reply cjs/node-mongo-helpers.js]: @samuko-things see what I changed `esm` template here, I've merged it in this your PR.
https://github.com/collabo-community/backend-api-boilerplates/pull/3/files

> Use the same for `ts` and `cjs` too

Comment (samuko-things) [reply cjs/node-mongo-helpers.js]: @Ifycode, I've made the changes and updated the cjs and ts file as requested.

**Review by Ifycode — APPROVED** [review]

**Pull request merged**